File: W3_T4/Person.java
public class Person {
    private int age;
    public String first_name;
    public String last_name;

    public Person(String fname, String lname, int age) {
        this.first_name = fname;
        this.last_name = lname;
        this.age = age;
    }

    public int getAge() {
        return this.age;
    }

    public void ageUp() {
        this.age++;
    }

    public String getFullname() {
        return this.first_name + " " + this.last_name;
    }

    public void printFullname() {
        System.out.println(this.first_name + " " + this.last_name);
    }
}

File: W3_T2/person.js
class Person {
    #age
    constructor(fname, lname, age) {
        this.first_name = fname;
        this.last_name = lname;
        this.#age = age;
    }

    getAge() {
        return this.#age;
    }

    ageUp() {
        this.#age++;
    }

    getFullname() {
        return `${this.first_name} ${this.last_name}`;
    }

    printFullname() {
        console.log(`${this.first_name} ${this.last_name}`);
    }
}

module.exports = Person;
